superset-frontend/webpack.config.js

```diff
@@ -214,34 +219,34 @@ const config = {
               'react',
               'react-dom',
               'prop-types',
+              'react-prop-types',
+              'prop-types-extra',
               'redux',
               'react-redux',
               'react-hot-loader',
               'react-select',
               'react-sortable-hoc',
               'react-virtualized',
               'react-table',
+              'react-ace',
               '@hot-loader.*',
               'webpack.*',
               '@?babel.*',
               'lodash.*',
               'antd',
               '@ant-design.*',
               '.*bootstrap',
+              'react-bootstrap-slider',
               'moment',
               'jquery',
               'core-js.*',
               '@emotion.*',
-              'd3.*',
+              'd3',
+              'd3-(array|color|scale||interpolateformat|selection|collection|time|time-format)',
```

Comment:
typo

`||interpolateformat` => `|interpolate|format`

Reply:
Wow, just wow... Thanks for spotting this!